app/src/main/java/com/hacktzi/blog/MainActivity.kt

```diff
@@ -5,6 +5,8 @@ import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
+        //Thread.sleep(10000)//show splashScreen 10 seconds
```

Comment:
Since this is the code for development, please try to do not send comments :)

Reply:
Ok, I will delete them :)